ansible-ipi-install/roles/installer/tasks/59_cleanup_bootstrap.yml

```diff
@@ -14,11 +14,19 @@
   become: yes
   tags: cleanup
 
+- name: Get list of all paused VMs
+  virt:
+    command: list_vms
+    state: paused
+  register: paused_vms
+  become: yes
+  tags: cleanup
+
 - name: Destroy old bootstrap VMs, if any
   virt:
     name: "{{ item }}"
     command: destroy
-  with_items: "{{ running_vms.list_vms }}"
+  with_items: "{{ running_vms.list_vms + paused_vms.list_vms}}"
```

Comment:
@p3ck Thanks for the PR! I have a question, while I know you mentioned virt.py doesn't do this well, I was going to ask would this work? 

My thinking here is that the running_vms is explicitly looking for just that, running VMs not paused so I'm wondering if using the all VMs would of been a better choice?

```
with_items: "{{ all_vms.list_vms }}"
```

Reply:
I'm guessing it will throw an error if we try and destroy and already destroyed vm?  Kind of a tricky one since I don't think we want to ignore_errors

Reply:
Ok, also could we make the with_items a list?

```
with_items: 
  - "{{ running_vms.list_vms }}"
  - "{{ paused_vms.list_vms }}"
```

I'll test this out tomorrow make sure all is good and merge it up.